themes/SuiteP/css/style.css

```diff
@@ -11205,4 +11205,85 @@ div#content div#pagecontent form#EditView div.edit.view table tbody tr td input[
         height: initial;
         padding: 0px 0px 15px 0px !important;
     }
+}
+
+#select_actions_disabled_top > a > label.hidden-mobile,
```

Comment:
Again, too long.  Refactor elements to have specific classes to refer to rather than having to many child selectors